openage/convert/processor/export/media_exporter.py

```diff
@@ -462,6 +465,10 @@ def _get_media_cache(
             from ...value_object.read.media.smx import SMX
             image = SMX(media_file.read())
 
+        elif source_file.suffix.lower() == ".sld":
+            from ...value_object.read.media.sld import SLD
+            image = SLD(media_file.read())
+
```

Comment:
While we're at it, can you add a final `else` that throws an error? :)

like here:

https://github.com/SFTtech/openage/blob/9e04789941defac9fe5c67cfdc3918e0f9e159bf/openage/convert/processor/export/media_exporter.py#L383-L386

Reply:
This would also be a good addition this line

https://github.com/SFTtech/openage/blob/9e04789941defac9fe5c67cfdc3918e0f9e159bf/openage/convert/processor/export/media_exporter.py#L227

Reply:
yes, please :)